packages/shared/src/components/checklist/actionUtils.ts

```diff
@@ -0,0 +1,24 @@
+import { Squad } from '../../graphql/sources';
+import { Action, ActionType } from '../../graphql/actions';
+
+// taken from DB default value
+const placeholderImage =
+  'https://daily-now-res.cloudinary.com/image/upload/s--LrHsyt2T--/f_auto/v1692632054/squad_placeholder_sfwkmj';
+
+export const getEditActions = (squad: Squad): Action[] => {
+  const isComplete =
+    !!squad.name &&
+    !!squad.description &&
+    !!squad.image &&
+    squad.image !== placeholderImage;
```

Comment:
Is this really mantadory for going public? 😅

Reply:
I would tend to agree that having an image on public squad is mandatory. When you look at the public squads list, you should be able to distinguish visually which one is which.